Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -251,10 +262,18 @@ void AuthenticatorCoordinator::discoverFromExternalSource(const Document& docume
     }
 
     if (requestOptions.signal) {
-        requestOptions.signal->addAlgorithm([weakThis = WeakPtr { *this }](JSC::JSValue) {
+        m_isCancelling = true;
```

Comment:
This isn't the point where we are cancelling, only that the RP passed an abort signal. This needs to go in the lambda passed to addAlgorithm below.

Reply:
ditto